src/script/util/KeyboardUtil.ts

```diff
@@ -83,40 +83,26 @@ export const isRemovalAction = (key: string): boolean => removalKeys.includes(ke
 
 export const isSpaceOrEnterKey = (key: string): boolean => key === KEY.SPACE || key === KEY.ENTER;
 
-type KeyboardHandler = (event: KeyboardEvent) => void;
-
-const escKeyHandlers: KeyboardHandler[] = [];
-
-document.addEventListener('keydown', event => {
```

Comment:
1. This doesn't even work
2. If that would work, we'd have this functionality unpredictably each time we import this file, makes no sense